src/Schema.php

```diff
@@ -25,4 +25,28 @@ public static function deactivate() {
   public static function uninstall() {
   }
 
+  /**
+   * Cron event callback to ensure proper database indexes.
+   */
+  public static function cron_ensure_back_in_stock() {
+    global $wpdb;
+    $ids = $wpdb->get_col($wpdb->prepare("SELECT ID FROM {$wpdb->posts}
+      INNER JOIN {$wpdb->postmeta} pm ON {$wpdb->posts}.ID = pm.post_id
+      LEFT JOIN {$wpdb->postmeta} moeve ON {$wpdb->posts}.ID = moeve.post_id AND moeve.meta_key LIKE %s
+      WHERE pm.meta_key = %s
+        AND pm.meta_value <= CURRENT_DATE()
+        AND moeve.meta_id IS NULL", [
+          '_woocommerce-moeve_id_%',
```

Comment:
Did you test and confirm that this works correctly?  To my knowledge, `wpdb::prepare()` treats percent signs specially as they are also the variable placeholders.  But maybe it works here because it appears at the end of string?

Reply:
@sun I did test and the result is correct, when I had it inside the sql string like:
```mysql
LEFT JOIN {$wpdb->postmeta} moeve ON {$wpdb->posts}.ID = moeve.post_id AND moeve.meta_key LIKE '_woocommerce-moeve_id_%'
```
then it would not work, but prepared like this it will return the same amount of results and the full raw query.